### PR TITLE
Setup SMB Server for sanity tests

### PR DESCRIPTION
**Pull request by boddumanohar**
**What type of PR is this?**

Fixes #

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [x] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

File: test/sanity/secrets.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+NodeStageVolumeSecret:
```
Comment (andyzhangx) [inline test/sanity/secrets.yaml:1]: what is this file for?

Comment (boddumanohar) [reply test/sanity/secrets.yaml:1]: this is the format to pass smb server secrets

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**